CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+**interBTC** is an open-source software project, providing a 1:1 Bitcoin-backed asset - fully collateralized, interoperable, and censorship-resistant.
+
+## Rules
+
+There are a few basic ground-rules for contributors (including the maintainer(s) of the project):
+
+- **Master** must have the latest changes and its history must never be re-written.
+- **Non-master branches** must be prefixed with the *type* of ongoing work.
+- **All modifications** must be made in a **pull-request** to solicit feedback from other contributors.
+- **All commits** must be **signed**, have a clear commit message to better track changes, and must follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary) standard.
+- **All tags** must follow [semantic versioning](https://semver.org/).
+
+## Workflow
+We use [Github Flow](https://guides.github.com/introduction/flow/index.html), so all code changes should happen through Pull Requests.
+
+## Issues
+We use GitHub issues to track feature requests and bugs.
+
+### Bug Reports
+Please provide as much detail as possible, see the GitHub templates if you are unsure.
+
+## Coding Style
```

Comment:
@tomjeatt please add any special things that we apply. I guess most is auto-check (or should be eventually).

Reply:
@nud3l yes, once the vault cards are done I'm going to make sure the code quality tools (es-lint, prettier) are working properly and applying all the rules automatically.

Reply:
I've added a note about Prettier/ESLint. Unless we run into problems with contributors I think that's enough detail, as the rules are enforced automatically. We use a pretty standard ruleset (especially with Prettier) so nothing out of the ordinary.